common/cosmwasm-smart-contracts/service-provider-directory/src/types.rs

```diff
@@ -7,28 +7,41 @@ use serde::{Deserialize, Serialize};
 /// The directory of services are indexed by [`ServiceId`].
 pub type ServiceId = u32;
 
+pub type IdentityKey = String;
```

Comment:
If you have to redefine our existing types, perhaps they should be moved to some contract-common crate instead?

Reply:
Yes, done!